Vagrantfile

```diff
@@ -0,0 +1,46 @@
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.network "forwarded_port", guest: 5000, host: 5000
+  config.vm.provision "shell", privileged: false, inline: <<-SHELL
+    sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
+    sudo apt-get install build-essential
+    
+    echo 'sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
+    libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev' >> ~/.profile
+    
+    git clone https://github.com/pyenv/pyenv.git ~/.pyenv
+
+    export PATH="$HOME/.pyenv/bin:$PATH"
+    eval "$(pyenv init --path)"
+    echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> ~/.profile
+    echo 'eval "$(pyenv init --path)"' >> ~/.profile
+
+    echo 'eval "$(pyenv init -)"' >> ~/.bashrc
+
+    gcc
+
+    sudo pyenv install 3.8.5
+
+    sudo pyenv global 3.8.5
+
+    sudo curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```

Comment:
Shouldn't need sudo.

Also I think you're missing a `-` at the end of the command. It's an option for the `python` command to tell it to execute code from stdin (the result of the `|`) instead of using it to open a python shell or execute a .py file.